ch_pipeline/processing/daily.py

```diff
@@ -364,56 +516,76 @@ class DailyProcessing(base.ProcessingType):
     default_params = {
         # Time range(s) to process
         "intervals": [
-            ## Priority days to reprocess
-            # Good ranges from rev_00
+            # Two short intervals either side of the caltime change (1878,
+            # 3000), one with the weird 4 baseline issue (1912), one with a
+            # single baseline-freq issue (1960), and one with a decorrelated
+            # cylinder (2325) in order to have a few good test days at the very
+            # start
+            {"start": "CSD1878", "end": "CSD1879"},
+            {"start": "CSD1912", "end": "CSD1912"},
```

Comment:
I thought you always need `"end"` = `"start"+1`, i.e.:
```suggestion
            {"start": "CSD1912", "end": "CSD1913"},
```
(Same comment for 1960 and 2325 below)

Reply:
For this CSD spec thing. I made the ends *inclusive*. https://github.com/chime-experiment/ch_pipeline/blob/master/ch_pipeline/processing/daily.py#L650

That may be confusing I guess. We can consider changing it in the future.

Reply:
Ah, okay. It would reduce confusion if there was a consistent convention for these single-day intervals in this file

Reply:
Ah, yeah, I can see that being confusing, although actually I did intend those to be two day intervals!

Reply:
Added a comment to clarify that.